app/views/happy_prices/_form.html.erb

```diff
@@ -0,0 +1,42 @@
+  <%= form_with(model: [@business, @item, @happy_price], local: true, method: method, class: "col-lg-6") do |form| %>
+
+  <!-- ERRORS !-->
+  <% if @happy_price.errors.any? %>
+    <div class="alert alert-warning" role="alert">
+      <p> Le produit n'a pas pu être créé pour les raisons suivantes :</p>
+      <ul>
+        <% @happy_price.errors.full_messages.each do |message| %>
+          <li> <%= message %> </li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <!-- price !-->
+  <div class="form-group">
+    <%= form.label :price, 'Prix' %>
+    <%= form.number_field :price, step: 0.01, min: 0.01, class: "form-control text-primary"  %>
+  </div>
+
+  <!-- quantity !-->
+  <div class="form-group">
+    <%= form.label :price, 'Quantité' %>
```

Comment:
Shouldn't it be "form.label :quantity" here ?

Reply:
Yes